docs/getting-started.md

```diff
@@ -6,10 +6,58 @@ nav_order: 2
 
 # Getting Started
 
-**Authelia** can be tested in a matter of seconds with Docker and docker-compose.
+## Docker Compose
 
-In order to deploy the current version of Authelia locally, run the following
-command and follow the instructions of bootstrap.sh:
+**Authelia** can be deployed as a local setup not requiring SQL server, Redis cluster
+or an LDAP server. In some cases, like protecting personal projects/websites, it can be
+fine to use this setup but beware that this setup is non-resilient to failures so it should
+be used at your own risk.
+
+The setup is called local since it reduces the number of components in the architecture to
+only two: a reverse proxy such as Nginx, Traefik or HAProxy and Authelia.
+
+Connection details to Redis are optional. If not provided, sessions will be stored in
+memory instead. This has the inconvenience of logging out users every time Authelia restarts.
+
+## Steps
+
+These commands are intended to be run sequentially:
+
+- `git clone https://github.com/authelia/authelia.git`
+- `cd authelia/compose/local`
+- `sudo ./setup.sh`
+- `docker-compose up -d`
+
+You can now visit the following locations; replace example.com with the domain you specified in the setup script:
+- https://public.example.com - Bypasses Authelia
+- https://traefik.example.com - Secured with Authelia one-factor authentication
+- https://secure.example.com - Secured with Authelia two-factor authentication (see note below)
+
+To visit https://secure.example.com you will need to register an OTP device, the email/link to generate your QR code will be in stored in: `./authelia/notification.txt`.
+Upon registering, you can grab this link easily by running the following command: `grep -Eo '"https://.*" ' ./authelia/notification.txt`.
+
+## Reverse Proxy
+
+Documentation for deploying a reverse proxy collaborating with Authelia is available
+[here](./supported-proxies/index.md).
+
+## FAQ
+
+### Can you give more details on why this is not suitable for production environments?
```

Comment:
This question should be in the FAQ of lite bundle only. You can remove it from the getting started.

Reply:
Given that there was only one question in the FAQ, just remove it.